README.md

```diff
@@ -52,6 +52,7 @@ Also a convenient [flat file](https://en.wikipedia.org/wiki/Flat-file_database)
     ├── srgan_train.ipynb/py (paired jupyter notebook/python script that trains the ESRGAN neural network model)
     └── test_ipynb.ipynb/py (paired jupyter notebook/python script that runs doctests in the other jupyter notebooks!)
 ```
+
```

Comment:
```suggestion
```
Not sure why there's an extra blank line here, if you can remove it that would be great!